cmd/webhook/main.go

```diff
@@ -58,9 +58,9 @@ func SharedMain(resourceHandlers map[schema.GroupVersionKind]webhook.GenericCRD)
 	if err != nil {
 		log.Fatal("Error parsing logging configuration:", err)
 	}
-	logger, atomicLevel := logging.NewLoggerFromConfig(config, component)
-	defer logger.Sync()
-	logger = logger.With(zap.String("cloud.google.com/events", component))
+	sl, atomicLevel := logging.NewLoggerFromConfig(config, component)
+	defer sl.Sync()
```

Comment:
Receive Adapter calls `metrics.FlushExporter()` as well as `sl.Sync`. Do we want to do that, here?

Reply:
Done.